tabs/configuration.js

```diff
@@ -140,13 +148,39 @@ TABS.configuration.initialize = function (callback, scrollPosition) {
 
     function load_name() {
         var next_callback = load_rx_config;
+
+        if (self.SHOW_OLD_BATTERY_CONFIG) {
+            next_callback = load_battery;
+        }
+
         if (semver.gte(CONFIG.apiVersion, "1.20.0")) {
             MSP.send_message(MSPCodes.MSP_NAME, false, false, next_callback);
         } else {
             next_callback();
         }
     }
 
+    // -----------------------------------------------
+    // Remove this when we officily retire BF 3.1.x
+    function load_battery() {
+        var next_callback = load_current;
+        if (semver.gte(CONFIG.flightControllerVersion, "3.1.0")) {
```

Comment:
Copy-paste this over from the code that has been removed when then new Power & Battery tab was created.